src/app/events/page.tsx

```diff
@@ -0,0 +1,308 @@
+'use client';
+
+import React, { useState, useEffect } from 'react';
+import { collection, getDocs } from 'firebase/firestore';
+import Calendar from 'react-calendar';
+import 'react-calendar/dist/Calendar.css';
+import './calendar.css';
+import {
+  TextInput,
+  Button,
+  Card,
+  Text,
+  Group,
+  Container,
+  Title,
+  Alert,
+  Box,
+} from '@mantine/core';
+import { useForm, isNotEmpty } from '@mantine/form';
+
+type EventFormValues = {
```

Comment:
Just a note, we can probably add a function for users to RSVP to an event. In that case we would just add an users: [] field to the event object